gmail/gmail-connector/gmail-connector-2.0.0/org.wso2.carbon.connector/repository/README.md

```diff
@@ -0,0 +1,59 @@
+ZOHO Invoice-Connector
```

Comment:
The README does not reflect what this module really contains. I believe, the term "ZOHO Invoice" has to be taken out and replaced with "Gmail Connector".